cmd/create/cluster/cmd.go

```diff
@@ -1354,7 +1354,7 @@ func run(cmd *cobra.Command, _ []string) {
 				r.Reporter.Errorf("%s", err)
 				os.Exit(1)
 			}
-			t := strings.Split(tag, ":")
+			t := strings.Split(tag, " ")
```

Comment:
Won't this break existing scripts people have? also, not sure the spaces would be properly handled by the flag parser.
perhaps instead we could only split the first colon, as i don't think colon in tag name is valid
```go
t := strings.SplitN(tag, ":", 2)

Reply:
Looks like colons and spaces are both permitted: https://docs.aws.amazon.com/tag-editor/latest/userguide/tagging.html#tag-conventions

Reply:
Yeah, thats why its on hold, need to get approval on this change. I have tested it locally and the spaces are handled as expected by the flag parser. 

If colons and spaces are both permitted then maybe we should move to something like `--tags key=foo,value=bar key=foo:bar, value=buzz`

Reply:
Only spaces represented in utf-8